src/Service/FileUploader.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace App\Service;
+
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class FileUploader
+{
+    private $targetDirectory;
+
+    public function __construct($targetDirectory)
+    {
+        $this->targetDirectory = $targetDirectory;
+    }
+
+    public function upload(UploadedFile $file)
+    {
+        $fileName = md5(uniqid()).'.'.$file->guessExtension();
+
+        try {
+            $file->move($this->getTargetDirectory(), $fileName);
+        } catch (FileException $e) {
+            // todo: handle exception if something happens during file upload
```

Comment:
@daria-sieroshtan 
Don't leave this code in this state
If you don't know what to do - better don't catch exception and leave your `todo` intact; in this case app will properly fail and monitoring will notify you